manifests/image-references

```diff
@@ -10,3 +10,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-hyperkube:v4.0
+  - name: cluster-policy-controller
+    from:
+      kind: DockerImage
+      name: quay.io/openshift/origin-cluster-policy-controller:v4.3
```

Comment:
@deads2k why this is 4.3, where the one above is 4.0? Not to mention the fact it's pointing to docker.io/... ?

Reply:
right, was also wondering that, thanks for reminding me